packages/manager/src/managers/SliceMachineManager.ts

```diff
@@ -0,0 +1,362 @@
+		// 	allStableVersions.map(async (version): Promise<PackageVersion> => {
+		// 		const releaseNotes =
+		// 			await this.versions.getSliceMachineReleaseNotesForVersion({
+		// 				version,
+		// 			});
+		//
+		// 		return {
+		// 			versionNumber: version,
+		// 			releaseNote: releaseNotes ?? null,
+		// 			// TODO
+		// 			kind: "MINOR",
+		// 		};
+		// 	}),
+		// );
+
+		// TODO: SM UI detects if a user is logged out by looking at
+		// `clientError`. Here, we simulate what the old core does by
+		// returning an `ErrorWithStatus`-like object if the user is
+		// not logged in.
+		const clientError: SliceMachineManagerGetStateReturnType["clientError"] =
+			profile ? undefined : { message: "Could not fetch slices", status: 401 };
+
+		return {
+			env: {
+				framework: "",
+				manifest: {
+					localSliceSimulatorURL: sliceMachineConfig.localSliceSimulatorURL,
+				},
+				mockConfig: {},
+				// TODO: Don't hardcode this!
+				packageManager: "npm",
+				// TODO: Don't hardcode this!
+				repo: sliceMachineConfig.repositoryName,
+				// TODO: Don't hardcode this!
+				sliceMachineAPIUrl: "http://localhost:9999",
+				intercomHash: profile?.intercomHash,
+				shortId: profile?.shortId,
+			},
+			libraries,
+			customTypes,
+			remoteCustomTypes,
+			remoteSlices,
+			clientError,
+		};
+	}
+
+	private async _getProfile(): Promise<PrismicUserProfile | undefined> {
+		let profile: PrismicUserProfile | undefined;
+
+		const isLoggedIn = await this.user.checkIsLoggedIn();
+
+		if (isLoggedIn) {
+			profile = await this.user.getProfile();
+			await this.user.refreshAuthenticationToken();
+		}
+
+		return profile;
+	}
+
+	private async _getLibraries(
+		sliceMachineConfig: SliceMachineConfig,
+	): Promise<SliceMachineManagerGetStateReturnType["libraries"]> {
+		const libraries: SliceMachineManagerGetStateReturnType["libraries"] = [];
+
+		if (sliceMachineConfig.libraries) {
+			await Promise.all(
+				sliceMachineConfig.libraries.map(async (libraryID) => {
+					const { sliceIDs } = await this.slices.readSliceLibrary({
+						libraryID,
+					});
+
+					if (sliceIDs) {
+						const components: SliceMachineManagerGetStateReturnType["libraries"][number]["components"] =
+							[];
+
+						await Promise.all(
+							sliceIDs.map(async (sliceID) => {
+								const [{ model }, { mocks }] = await Promise.all([
+									this.slices.readSlice({ libraryID, sliceID }),
+									this.slices.readSliceMocks({ libraryID, sliceID }),
+								]);
+
+								if (model) {
+									const screenshots: typeof components[number]["screenshots"] =
+										{};
+									await Promise.all(
+										model.variations.map(async (variation) => {
+											const screenshot = await this.slices.readSliceScreenshot({
+												libraryID,
+												sliceID,
+												variationID: variation.id,
+											});
+
+											if (screenshot.data) {
+												screenshots[variation.id] = {
+													path: "__stub__",
```

Comment:
Thanks for catching that. `__stub__` was added because `path` is no longer needed. It was originally used by `slice-machine-ui`'s server since it managed saving and updating Slice screenshots. Now that `@slicemachine/manager` does it transparently though plugins, we can remove `path`.

It wasn't removed because `slice-machine-ui` has code that depends on it, such as the `ScreenshotUI` interface: https://github.com/prismicio/slice-machine/blob/b68e362768d39ed5f693c6e00162eaf3d58bcf92/packages/slice-machine/lib/models/common/ComponentUI.ts#L18

I think it's safe to remove it, but I was trying to touch as little of SM UI's code as possible.